src/igl/opengl/RenderCommandEncoder.cpp

```diff
@@ -55,6 +55,18 @@ int toGlType(IndexFormat format) {
   IGL_UNREACHABLE_RETURN(GL_UNSIGNED_INT)
 }
 
+uint8_t getIndexByteSize(int indexType) {
```

Comment:
`GLenum`?

Reply:
"RenderCommandEncoder::indexType_" is  int type, so i  keep it consistent with "indexType_".

Reply:
If necessary, I can modify both indexType and RenderCommandEncoder::indexType_ together.

Reply:
Use GLenum better

Reply:
done